examples/Dockerfile.centos8

```diff
@@ -0,0 +1,6 @@
+# ch-test-scope: standard
+FROM centos:8
+# Install the dnf ovl plugin to workaround rpmdb corruptions issues when using
+# overlayfs as the storage driver.
+RUN dnf -y install https://download.fedoraproject.org/pub/epel/playground/8/Everything/x86_64/os/Packages/d/dnf-plugin-ovl-0.0.3-1.epel8.playground.noarch.rpm
```

Comment:
Is this in EPEL and we can install it from there? Generally speaking, EPEL is super useful, so why don't we install it first.

Reply:
It is in EPEL in the "epel-playground" repository. Since up until the lustre tests we didn't pull anything from EPEL I didn't want to increase the size of resulting images for something we weren't using. If you feel it's good to have regardless I'll install it.

Reply:
I think we should just go ahead and enable EPEL and PowerTools as one of the first steps. CentOS is way more usable with it.

That said, does `dnf` work well enough on overlayfs to enable EPEL and install that plugin?